vite.config.mjs

```diff
@@ -7,6 +7,7 @@ export default defineConfig({
   base: "/",
   plugins: [react(), svgLoader()],
   server: {
+    allowedHosts: ['aapodwalk.local.itkdev.dk'],
```

Comment:
Will this also work in production, i.e. on another domain?

Reply:
I added the solution directly in vite.config.mjs, and added a todo for the improvement where the value could be passed via .env.